recipes/argon2/all/conanfile.py

```diff
@@ -0,0 +1,62 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class Argon2Conan(ConanFile):
```

Comment:
for `tools.remove_files_by_mask`

```suggestion

required_conan_version = ">=1.29.1"


class Argon2Conan(ConanFile):
```